openmetadata-ui/src/main/resources/ui/src/axiosAPIs/searchAPI.test.ts

```diff
@@ -0,0 +1,300 @@
+              version: 0.1,
+              updatedAt: 1661336540995,
+              updatedBy: 'anonymous',
+              email: 'aaron_johnson0@gmail.com',
+              href: 'http://localhost:8585/api/v1/users/2cae227c-e2c4-487c-b52c-a96ae242d90d',
+              isAdmin: false,
+              deleted: false,
+              roles: [],
+              inheritedRoles: [],
+              entityType: 'user',
+              teams: null,
+              some: {
+                nested: {
+                  nullValue: null,
+                },
+              },
+            },
+          },
+        ],
+      },
+    ],
+  },
+};
+/* eslint-enable @typescript-eslint/camelcase */
+
+describe('searchAPI tests', () => {
+  beforeEach(() => jest.resetModules());
+
+  it('suggestQuery should return object and aggregations', async () => {
+    jest.mock('./index.js', () => ({
+      get: jest
+        .fn()
+        .mockImplementation(() =>
+          Promise.resolve({ data: mockTableSearchResponse })
+        ),
+    }));
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { searchQuery } = require('./searchAPI');
+    const res = await searchQuery({ searchIndex: SearchIndex.TABLE });
+
+    expect(res.hits.total.value).toEqual(10_000);
+
+    expect(res.hits.hits.length).toEqual(1);
+    expect(res.hits.hits[0]._index).toEqual(SearchIndex.TABLE);
+    expect(res.hits.hits[0]._source).toEqual(
+      expect.objectContaining({
+        id: '9b30a945-239a-4cb7-93b0-f1b7425aed41',
+        name: 'raw_product_catalog',
+        fullyQualifiedName:
+          'sample_data.ecommerce_db.shopify.raw_product_catalog',
+        description:
+          'This is a raw product catalog table contains the product listing, price, seller etc.. represented in our online DB. ',
+        version: 0.1,
+        updatedAt: 1661336543968,
+        updatedBy: 'anonymous',
+        href: 'http://localhost:8585/api/v1/tables/9b30a945-239a-4cb7-93b0-f1b7425aed41',
+        tableType: 'Regular',
+      })
+    );
+
+    expect(res.aggregations).toEqual(
+      expect.objectContaining({
+        EntityType: {
+          buckets: expect.arrayContaining([
+            {
+              key: 'table',
+              /* eslint-disable-next-line @typescript-eslint/camelcase */
+              doc_count: 10960,
+            },
+          ]),
+        },
+        ServiceName: {
+          buckets: expect.arrayContaining([
+            {
+              key: 'trino',
+              /* eslint-disable-next-line @typescript-eslint/camelcase */
+              doc_count: 10924,
+            },
+            {
+              key: 'sample_data',
+              /* eslint-disable-next-line @typescript-eslint/camelcase */
+              doc_count: 36,
+            },
+          ]),
+        },
+        Tags: {
+          buckets: [],
+        },
+      })
+    );
+  });
+
+  it('searchQuery should not return nulls', async () => {
+    afterEach(() => jest.clearAllMocks());
```

Comment:
This is not required
It's set globally [here](https://github.com/open-metadata/OpenMetadata/blob/b914afcb6ac4767496a68e8bdcfa493237dbb7c5/openmetadata-ui/src/main/resources/ui/jest.config.js#L45)